components/search/MoreFacetsDropdownOptions.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <b-dropdown-form>
+    <strong
+      class="mb-4 d-inline-block"
+    >
+      {{ $t(`facets.${facet.name}`).name }}
+    </strong>
+    <b-form-checkbox-group
+      class="option-group"
+      plain
+    >
+      <b-form-checkbox
+        v-for="(filter, i) in [].concat(facet.fields).splice(0, limitTo)"
+        :id="`${$t(`facets.${facet.name}`).name} - ${filter.label}`"
+        :key="i"
+        :value="filter.label"
+        :name="filter.label"
+        :data-qa="`${$t(`facets.${facet.name}`).name} checkbox`"
+        class="mb-3"
+      >
+        {{ filter.label }}
+        <span
+          class="reset icon-close"
+          :aria-label="$t('facets.button.reset')"
+        />
+      </b-form-checkbox>
+      <div
```

Comment:
I think we can make this a `<template>` and remove the `class` attribute.

Reply:
breaks if i remove the class and if i change it to template